setup.md

```diff
@@ -50,16 +50,16 @@ In the lesson, you will find out how to access the data in this folder.
 >
 > Other solutions are available for running Bash commands on Windows. 
 > There is now a Bash shell command-line tool available for Windows 10. 
-> Additionally, you can run Bash commands on a remote UNIX computer or server from your Windows machine. 
-> This can be done through a Secure Shell (SSH) client. 
+> Additionally, you can run Bash commands on a remote computer or server that already has a Unix Shell, from your Windows machine. 
+> This can usually be done through a Secure Shell (SSH) client. 
```

Comment:
Why usually? I realize that the remote computer needs to accessible (network), and running an SSH server, but do we really want to explain this?